target/linux/ipq40xx/config-4.19

```diff
@@ -66,6 +66,7 @@ CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_MQ_PCI=y
 CONFIG_BOUNCE=y
 # CONFIG_CACHE_L2X0 is not set
+CONFIG_CC_HAS_ASM_GOTO=y
```

Comment:
This is runtime option, so it shouldn't be set.

Reply:
I did not set it manually, it was set by make kernel_menuconfig when I was enabling SDHCI and LDO driver.
I have also seen it on other targets being enabled lately.

Reply:
>  I have also seen it on other targets being enabled lately.

It doesn't mean, that it's correct.

Reply:
I can manually remove it then, it looks like simple kernel config refresh will enable it.

Reply:
Yes, we need to [live with that](https://bugs.openwrt.org/index.php?do=details&task_id=2588) or fix our tools :-)